### ISSUE TITLE
Request for additional kernel modules

### ISSUE THREAD

**Issue by pillarama** [thaneb/TL-MR3020-OpenWRT-Extroot-Ready#12]
Hi! This image was exactly what I needed - thanks! 
However I need to run a Huawei e8372 LTE modem on my MR3020, but some required kernel modules don't seem to be available in the repo. I think I would need kmod-usb-net kmod-usb-net-cdc-ether (usbnet and cdc_ether respectively) is there any chance of having those added please?

Thanks!

**Comment by thaneb** [comment]
Hi there!
I will have a look at it this week-end, strange because I thought to have compiled everything I could, including these kernel mods (as I am also concerned about it for some setup).

**Comment by thaneb** [comment]
Hi pillarama

Great news, I have finally been able to boot my dedicated VM, and guess what? I successfully compiled all the USB kernel modules available! The updated package and package list have been uploaded to the server, so you should be able to go ahead now. Sorry for the delay, my computer wasn't booting..

**Comment by pillarama** [comment]
Hi thaneb
 No need to apologize! I'm just grateful that you have built this at all let alone continue to give your time to support it. 
I gave it a go this morning and while I did have to do a sysupgrade to the new image because the kernel fingerprint changed and then ended up having to rebuild the extroot (which might have been due to mistakes I made), after this is worked perfectly and my USB modem is now up and running!  None of that was a big deal for me as I hadn't customised it that much and could just backup and restore the config.

Thanks so much! I really appreciated it.
